symbols/arm7.yml

```diff
@@ -79,10 +133,46 @@ arm7:
         EU: 0x2385F40
         NA: 0x2385F40
         JP: 0x2385F40
+        EU-WRAM: 0x37FDD58
+        NA-WRAM: 0x37FDD58
+        JP-WRAM: 0x37FDD58
       description: |-
         Gets the processor's current operating mode.
-        
+
         See https://problemkaputt.de/gbatek.htm#armcpuflagsconditionfieldcond
         
         return: cpsr & 0x1f (the cpsr mode bits M4-M0)
+    - name: __divsi3
+      address:
+        EU-WRAM: 0x3806BC8
+        NA-WRAM: 0x3806BC8
+        JP-WRAM: 0x3806BC8
+      description: |-
+        Copy of the ARM9 function. See arm9.yml for more information.
+        
+        r0: dividend
+        r1: divisor
+        return: (quotient) | (remainder << 32)
+    - name: __udivsi3
+      address:
+        EU-WRAM: 0x3806DD4
+        NA-WRAM: 0x3806DD4
+        JP-WRAM: 0x3806DD4
+      description: |-
+        Copy of the ARM9 function. See arm9.yml for more information.
+        
+        r0: dividend
+        r1: divisor
+        return: (quotient) | (remainder << 32)
+    - name: __udivsi3_no_zero_check
+      address:
+        EU-WRAM: 0x3806DDC
+        NA-WRAM: 0x3806DDC
+        JP-WRAM: 0x3806DDC
+      description: |-
+        Copy of the ARM9 function. See arm9.yml for more information.
+        
+        r0: dividend
+        r1: divisor
+        return: (quotient) | (remainder << 32)
```

Comment:
It looks like these exist in the binary? Can we add the EU/NA/JP offsets?

Reply:
Yeah I still have to calculate these since Ghidra only shows me WRAM addresses, hold on.

Reply:
Added